From 7162c327f9a15b2864095d083f16de4c7a17ffbd Mon Sep 17 00:00:00 2001
From: Jean-Philippe Brucker <jean-philippe@linaro.org>
Date: Fri, 18 Feb 2022 12:15:35 +0000
Subject: [PATCH 68/97] iommu/arm-smmu-v3: Add a few missing bits

The KVM SMMUv3 driver will need more configuration bits for stream table
entries.

Signed-off-by: Jean-Philippe Brucker <jean-philippe@linaro.org>
---
 include/linux/arm_smmu_v3_regs.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/linux/arm_smmu_v3_regs.h b/include/linux/arm_smmu_v3_regs.h
index a8a74f736d8c..2d7369112ce4 100644
--- a/include/linux/arm_smmu_v3_regs.h
+++ b/include/linux/arm_smmu_v3_regs.h
@@ -226,7 +226,10 @@
 #define STRTAB_STE_2_VTCR_S2PS		GENMASK_ULL(18, 16)
 #define STRTAB_STE_2_S2AA64		(1UL << 51)
 #define STRTAB_STE_2_S2ENDI		(1UL << 52)
+#define STRTAB_STE_2_S2AFFD		(1UL << 53)
 #define STRTAB_STE_2_S2PTW		(1UL << 54)
+#define STRTAB_STE_2_S2HD		(1UL << 55)
+#define STRTAB_STE_2_S2HA		(1UL << 56)
 #define STRTAB_STE_2_S2R		(1UL << 58)
 
 #define STRTAB_STE_3_S2TTB_MASK		GENMASK_ULL(51, 4)
-- 
2.34.1

